src/SFA.DAS.ProviderRequestApprenticeTraining.Web/Views/EmployerRequest/SelectProviderPhoneNumber.cshtml

```diff
@@ -49,7 +49,7 @@
                             @foreach (var phone in Model.PhoneNumbers)
                             {
                                 <div class="govuk-radios__item">
-                                    <input class="govuk-radios__input" type="radio" asp-for="SelectedPhoneNumber" value="@phone" >
+                                    <input class="govuk-radios__input" type="radio" asp-for="SelectedPhoneNumber" value="@phone" id="@phone" >
```

Comment:
It has not been reported as a bug yet - but this needs to use the id of the model property for the first radio so that the error message can jump to it - like has been done for the SelectedRequests